src/Miner/Miner.h

```diff
@@ -18,39 +18,39 @@
 
 #include "Logging/LoggerRef.h"
 
-namespace CryptoNote {
+namespace cn {
 
 struct BlockMiningParameters {
   Block blockTemplate;
   difficulty_type difficulty;
 };
 
-class Miner {
+class miner {
```

Comment:
The class should remain in CamelCase here
```suggestion
class Miner {
```